mlflow/utils/databricks_utils.py

```diff
@@ -134,6 +134,18 @@ def get_notebook_path():
         return _get_extra_context("notebook_path")
 
 
+def get_databricks_runtime():
+    try:
+        spark_session = _get_active_spark_session()
```

Comment:
I think this function may need a little reworking, since `_get_active_spark_session` will return a Spark session for any user who has pyspark installed in their environment. However, not all Spark sessions will have this spark tag set. We may be better off using util functions like `is_in_databricks_notebook` and `is_in_databricks_job` to gate this functionality behind the specific scenarios in which we expect this tag to be set.

Reply:
Also, that way, we don't backdoor this code into folks using the client in non-Databricks settings

Reply:
@sueann jobs/notebooks are the only scenarios right?

Reply:
yeah that should be it